common/k8stest/util_apps.go

```diff
@@ -419,6 +487,31 @@ func (dfa *FioApp) Cleanup() error {
 				return fmt.Errorf("verify replicas before deleting PVC failed: %v", err)
 			}
 		}
+	} else {
+		err := DeleteDeployment(dfa.DeployName, common.NSDefault)
+		if err != nil {
+			return fmt.Errorf("failed to delete fio deployment %s, err: %v", dfa.DeployName, err)
+		}
+		labels := "e2e-test=fio"
```

Comment:
don't hardcode labels otherwise it might create problem if somebody uses different labels while creating fio deployment

Reply:
Removed